e2e_tests/docker_image/Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN rm -rf $ibmtpm_name/src $ibmtpm_name.tar.gz
 WORKDIR /tmp
 RUN git clone https://github.com/opendnssec/SoftHSMv2.git \
 	&& cd SoftHSMv2 \
-	&& git reset --hard 20a53bd083a6134ce2230f80edda5dc8be0366bd
+	&& git reset --hard 35938595f83923504751b40535570342f706a634
```

Comment:
Why don't we use an archive instead? Then it would be clear what version of the libraries we use.

Reply:
You mean the zipped code attached to a tag on the repo? Mostly because they've not tagged a version in a while and there are a few fixes that have been merged since that help with the problem we usually encounter. I guess we should update [the book](https://parallaxsecond.github.io/parsec-book/parsec_service/tests/index.html?highlight=soft#testing-the-pkcs11-provider-using-the-software-hsm) to reflect this version change.

Reply:
We can also ping them to ask for a new tag

Reply:
In softhsm recipe in meta-openembedded layer they use archives from https://dist.opendnssec.org/source/
The last one is 2.6.1
I don't know how up to date the archives there though
That was just an idea if tags are not updated in the repo. You're welcome to ignore it.

Reply:
Hmm, is this for a test version of the service, in Yocto?
2.6.1 is indeed the latest tag on the repo, but it's about a year old now.

Reply:
Yes, it is used for manual testing only.

Reply:
For manual testing 2.6.1 should be good - the version we're pulling here for our CI has some extra fixes, but there's nothing new in terms of functionality.